Ui/Component/Listing/Columns/Raw.php

```diff
@@ -31,6 +31,23 @@ public function __construct(
         $this->escaper = $escaper;
     }
 
+    /**
+     * @param $message
+     * @return string|string[]|null
+     */
+    private function hideConfidentialInfo(string $message): string
+    {
+        $isHideConfidentialInfo = true;
+        if ($isHideConfidentialInfo) {
```

Comment:
@dmitry-mironov do we even need a conditional statement here? it looks like we have an unnecessary variable + IF statement